ChangeLog

```diff
@@ -22,6 +22,9 @@ Features
      MBEDTLS_CMAC_ALT). Submitted by Steve Cooreman, Silicon Labs.
    * Add support for alternative implementations of GCM, selected by the
      configuration flag MBEDTLS_GCM_ALT.
+   * In TLS servers, support offloading private key operations to an external
```

Comment:
I disagree. To me, “enable” conveys “changed a setting to allow” more than “add code that makes it possible”. The existing entries tend to use “support”, not “enable”. This entry doesn't claim to add support for external cryptoprocessors, it claims to add support for “offloading …”, which is correct. I'm open to suggestions if you find this unclear.

Reply:
Just "supporting offloading" alone makes sense, but when the sentence says "supporting offloading to an external device" this could be parsed to imply support for a device, but one can parse it either way. For me, "enable" can convey adding code to make it possible, and this doesn't imply at all that it's just from a setting change.

I don't have too strong an opinion either way here, and consistency with other messages has some benefit.